### DISCUSSION TITLE
Whitelist4QQ无法使用

### DISCUSSION THREAD

**Discussion by ShanZheY** [DreamVoid/MiraiMC#186] [问答 / Q&A] [answered]
Whitelist4QQ的配置文件都调过了，群号和机器人号也没输入错误，服务器后台miraimc能看到qq群发的消息，但是发送“申请白名单+ID”机器人没有任何回应（前缀没改）

**Comment by DreamVoid** [comment, marked as answer]
机器人被腾讯屏蔽了，一般为异地登录导致，暂时无解
